eng/pipelines/common/templates/run-test-job.yml

```diff
@@ -200,7 +200,12 @@ jobs:
           artifactName: '$(coreClrProductArtifactName)'
           displayName: 'CoreCLR product download for Mono'
 
-      - script: msbuild $(monoRepoRoot)/mono.proj -target:PatchCoreClrCoreRoot /p:Configuration=Release
+      - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt)
+		-ci
+                -projects $(Build.SourcesDirectory)/src/mono
```

Comment:
missing the .proj in the path? Is this `$(Build.SourcesDirectory)/src/mono/mono.proj`?